content/en/docs/integrations/protobuf.md

```diff
@@ -109,6 +109,11 @@ resides in
 can be converted to CUE using the following Go code
 
 {{< highlight go >}}
+  import (
+  	"cuelang.org/go/cue/format"
+  	"cuelang.org/go/encoding/protobuf"
+  )
+
```

Comment:
Following pattern of highlighting ["notable" cuelang.org packages](https://github.com/cuelang/cuelang.org/blob/42bcb4ee6fb146b64d98005a3f77c66e0ae8cb71/content/en/docs/integrations/openapi.md?plain=1#L27-L28) in the given code snippet rather than providing a strictly complete listing of all import statement used in the program.

Reply:
@dylankb apologies for the delay in replying here.

The problem however is that the code snipper that follows is not valid outside of a function. So I think we need some additional context (i.e. wrap in a simple `func main`) for what follows possibly with `// ...` to indicate this is an incomplete snippet.

Reply:
@myitcv Makes sense, made the suggested changes. Let me know what you think.